todo_app/templates/index.html

```diff
@@ -15,7 +15,7 @@ <h2>Items</h2>
           <input class="todo-text" type="text" name="title"><button type="submit" name="Add" value="Add">Add</button>
         </form>
 
-        {% for item in items%}
+        {% for item in view_model.items%}
           <li><span class="todo-text" >{{item.status}}: {{item.title}}</span><form action="/completeitem/{{item.id}}" method="POST" style="display: inline;"><button type="submit">Complete</button></form></li>
         {%endfor%}
```

Comment:
Minor, but worth noting that having not changed how we use our view model we're testing some functionality that our app ignores entirely so beyond practice it doesn't have any real value! One way we might take advantage of this logic is to hide the "Complete" buttons for items that are already completed - giving us something like the below:
```suggestion
        {% for item in view_model.todo_items%}
          <li><span class="todo-text" >{{item.status}}: {{item.title}}</span><form action="/completeitem/{{item.id}}" method="POST" style="display: inline;"><button type="submit">Complete</button></form></li>
        {%endfor%}
        {% for item in view_model.done_items%}
          <li><span class="todo-text" >{{item.status}}: {{item.title}}</span>
        {%endfor%}
```